addons/payment/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,48 @@
+---Fields in module 'payment'---
+payment      / payment.acquirer         / auto_confirm (selection)      : DEL required: required, selection_keys: ['authorize', 'confirm_so', 'generate_and_pay_invoice', 'none'], req_default: function
+payment      / payment.acquirer         / capture_manually (boolean)    : NEW 
+payment      / payment.acquirer         / country_ids (many2many)       : NEW relation: res.country
+payment      / payment.acquirer         / payment_flow (selection)      : NEW required: required, selection_keys: ['form', 's2s'], req_default: function
+payment      / payment.acquirer         / payment_icon_ids (many2many)  : NEW relation: payment.icon
```

Comment:
Not sure if that is useful. It will likely lead to duplicates. If people customized this, they can add the same image again.

Reply:
It is, Stefan! Don't make the customer to put again their specific icons for the payments. I have several with custom ones.

Reply:
@pedrobaeza i didn't get it, payment.icon seems to store icon's cards, and payment.acquirer icon didn't change(stored as ir.attachment) so it will be stored after migration even if there is custom ones.

Reply:
You are right. I have been confused with the name. Nothing to do indeed.